test/rule_test.dart

```diff
@@ -241,6 +241,9 @@ testRules(String ruleDir, {String analysisOptions}) {
 testRule(String ruleName, File file,
     {bool debug = false, String analysisOptions}) {
   registerLintRules();
+  if (ruleName != 'unnecessary_await_in_return') {
+    return;
```

Comment:
Should there be a comment here about why this test is being skipped?

Reply:
No, it should not be here.
I forgot to revert after debugging the failure.
Will do before landing.

Reply:
Done.